flips: add page (#8942)
tldr-pages/tldr: pages/common/flips.md

# flips

> > Flips Applies and creates IPS and BPS patches.

Reviewer: Perhaps this better
```suggestion
> Create and apply patches for IPS and BPS files.
```

> > More information: <https://www.smwcentral.net/?a=details&id=11474&p=section>.

Reviewer: I think the source code is more useful
```suggestion
> More information: <https://github.com/Alcaro/Flips>.
```

> - Open to create and apply patches interactively:

Reviewer: Perhaps this better
```suggestion
- Start Flips to create and apply patches interactively:
```

> - Open user interface to create/apply patch:

Reviewer: ```suggestion
- Open to create and apply patches interactively:
```

`flips`

> - Apply patch by creating new file:

Reviewer: Perhaps this better
```suggestion
- Apply a patch and create a new ROM file:
```

`flips --apply {{patch.bps}} {{rom.smc}} {{hack.smc}}`

> - Create patch from two ROMs:

Reviewer 2: ```suggestion
- Create a patch from two ROMs:
```

> - Create patch from two roms:

Reviewer: ```suggestion
- Create patch from two ROMs:
```
